dev/sparktestsupport/modules.py

```diff
@@ -777,6 +777,68 @@ def __hash__(self):
         "pyspark.ml.connect.functions",
         # ml unittests
         "pyspark.ml.tests.connect.test_connect_function",
+        # pandas-on-Spark unittests
```

Comment:
just to confirm, the doctests are not reused for now?

Reply:
Yes, that will be added in the subsequent task in SPARK-43003 right after this PR.

For now, I want to focus on validating the basic functionalities through unit tests and linters in the initial PR.